scripts/zkai_d64_native_relation_witness_oracle.py

```
@@ -0,0 +1,568 @@
+            "not full transformer inference",
+        ],
+        "next_backend_step": "encode this relation oracle as native Stwo AIR/export rows that consume the same public instance",
+    }
+    if mutation_suite is None:
+        payload.pop("mutation_suite")
+    return payload
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
+    if not isinstance(payload, dict):
+        raise NativeRelationWitnessOracleError("payload must be an object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)